ablator/modules/optimizer.py

```diff
@@ -12,6 +12,36 @@
 def get_parameter_names(model: torch.nn.Module, forbidden_layer_types: list[type]):
     """
     Returns the names of the model parameters that are not inside a forbidden layer.
```

Comment:
this is incorrect

Reply:
How about this: "Recurse into the module and return parameter names of all submodules, excluding modules that are of any type defined in `forbidden_layer_types`."

For Return section: "The names of the parameters with the following format: `<submodule-name>.<parameter-name>`."